master/getting-started/bare-metal/bare-metal.md

```diff
@@ -427,3 +430,280 @@ an ingress rule allowing access *to* port 999 and an egress rule allowing
+
+In addition to being applied before any DNAT, the enforcement of pre-DNAT
+policy differs from that of normal host endpoint policy in three key details,
+reflecting that it is designed for the policing of incoming traffic from
+outside the cluster:
+
+1. Only the ingress rules of a pre-DNAT policy are enforced.  If the policy has
+   any egress rules defined, they will have no effect.  (Standard connection
+   tracking is sufficient to allow return path traffic.)
+
+2. Pre-DNAT policy is enforced for all traffic arriving through a host
+   endpoint, regardless of where that traffic is going, and - in particular -
+   even if that traffic is routed to a local workload on the same host.
+   (Whereas normal host endpoint policy is skipped, for traffic going to a
+   local workload.)
+
+3. There is no 'default drop' semantic for pre-DNAT policy (as there is for
+   normal host endpoint policy).  In other words, if a host endpoint is defined
+   but has no pre-DNAT policies that explicitly allow or deny a particular
+   incoming packet, that packet is allowed to continue on its way, and will
+   then be accepted or dropped according to workload policy - if it is going to
+   a local workload - or to normal host endpoint policy - if not.
+
+## When do host endpoint policies apply?
+
+As stated above, normal host endpoint policies apply to traffic that arrives on
+and/or is sent to a host interface, except if that traffic comes from or is
+destined for a workload on the same host; but the rules for applying untracked
+and pre-DNAT policies are different in some cases.  Here we present and
+summarize all of those rules together, for all possible flows and all types of
+host endpoints policy.
+
+For packets that arrive on a host interface and are destined for a local
+workload - i.e. a locally-hosted pod, container or VM:
+
+- Untracked policies technically do apply, but never have any net positive
+  effect for such flows.
+
+  > **NOTE**
+  >
+  > To be precise, untracked policy for the incoming host interface may apply
+  > in the forwards direction, and if so it will have the effect of forwarding
+  > the packet to the workload without any connection tracking.  But then, in
+  > the reverse direction, there will be no conntrack state for the return
+  > packets to match, and there is no application of any egress rules that may
+  > be defined by the untracked policy - so unless the workload's policy
+  > specifically allows the relevant source IP, the return packet will be
+  > dropped.  That is the same overall result as if there was no untracked
+  > policy at all, so in practice it is as if untracked policies do not apply
+  > to this flow.
+
+- Pre-DNAT policies apply.
+
+- Normal policies do not apply - by design, because Calico enforces the
+  destination workload's ingress policy in this case.
+
+For packets that arrive on a host interface and are destined for a local
+server in the host namespace:
```

Comment:
Would it be better to say `server process`, just to avoid the reading parsing it as `server machine` (as I did and had to backtrack)